pkg/compose/watch.go

```diff
@@ -159,17 +170,15 @@ func (s *composeService) Watch(ctx context.Context, project *types.Project, serv
 			return err
 		}
 		watching = true
-
 		eg.Go(func() error {
 			defer watcher.Close() //nolint:errcheck
 			return s.watch(ctx, project, service.Name, options, watcher, syncer, config.Watch)
 		})
 	}
-
 	if !watching {
 		return fmt.Errorf("none of the selected services is configured for watch, consider setting an 'develop' section")
 	}
-	options.LogTo.Log(api.WatchLogger, "watch enabled")
+	options.LogTo.Log(api.WatchLogger, "Watch started")
```

Comment:
I personally preferred "enabled", but that's just me

Reply:
I agree, even the navigation menu message switches between `watch enabled/disabled`